main.sh

```diff
@@ -41,6 +41,18 @@ DAEMON=0
 STARTUP=0
 SYNC_GATEWAY=0
 WAIT=0
+NO_CLUSTER=0
+DO_CLUSTER=0
+DEFAULT_SERVICES=("data" "index" "analytics" "eventing" "fts" "query")
+SERVICES="data,index,analytics,eventing,fts,query"
+DATA_QUOTA=$(__getTotalRam)
+DATA_QUOTA=$((DATA_QUOTA / 2)) #50% of available as default
+INDEX_QUOTA=$(__getTotalRam)
+INDEX_QUOTA=$((15 * INDEX_QUOTA / 100 )) #15% of available as default
+SEARCH_QUOTA=256
```

Comment:
Do these not do percentages? Just wondering why the others are calculated.
Be good to have a comment and units.

Reply:
Yeah.. we don't do percentages.